hadoop-hdds/rocksdb-checkpoint-differ/src/main/java/org/apache/ozone/rocksdb/util/ManagedSstFileReader.java

```diff
@@ -216,7 +215,8 @@ protected abstract ClosableIterator<T> getKeyIteratorForFile(String file)
     public boolean hasNext() {
       try {
         do {
-          if (currentFileIterator.hasNext()) {
+          if (!Objects.isNull(currentFileIterator) &&
```

Comment:
nit: no need to use double negation.
```suggestion
          if (Objects.nonNull(currentFileIterator) &&
```

Reply:
done